miri-script/src/util.rs

```diff
@@ -143,7 +173,7 @@ impl MiriEnv {
         let MiriEnv { toolchain, cargo_extra_flags, .. } = self;
         cmd!(
             self.sh,
-            "cargo +{toolchain} test {cargo_extra_flags...} --manifest-path {manifest_path} {args...}"
+            "cargo +{toolchain} test {cargo_extra_flags...} --manifest-path {manifest_path} -- {args...}"
```

Comment:
This should probably be done by the caller, not here. Here it's somewhat inconsistent.

Then the docs will also need adjusting, and `ci.sh` as well... seems like passing two `--` actually works but it shouldn't be necessary. The README also needs updating.